R-package/R/xgb.config.R

```diff
@@ -0,0 +1,38 @@
+#' Global configuration consists of a collection of parameters that can be applied in the global
+#' scope. See \url{https://xgboost.readthedocs.io/en/latest/parameter.html} for the full list of
```

Comment:
Can we use `stable` instead of `latest`?

Reply:
Sure. Go ahead.